### DISCUSSION TITLE
Is there any plan to migrate to Bulma 1?

### DISCUSSION THREAD

**Discussion by kaangokdemir** [buefy/buefy#4036] [Q&A] [answered]
Hello,

I noticed that the new version of Bulma 1.0.0 has been released for some weeks now . ->[1.0.0](https://github.com/jgthms/bulma/releases/tag/1.0.0)

I'm curious if there are any plans to integrate this latest version of Bulma with Buefy.

Thank you.

**Comment by wesdevpro** [comment, marked as answer]
Hey @kaangokdemir! Thank you for your question. @kikuomax and I do not have any plans to continue creating new features for Buefy (Vue 2) because we are focusing on migrating from Vue2 to Vue3 ([Buefy-next](https://github.com/ntohq/buefy-next)). 

Please take some time to read this [Issue](https://github.com/buefy/buefy/issues/3874) 

Now to answer your question. No there is no official plan to migrate Buefy (Vue2) to Bulma v1. However some time after we pull all the changes from @ntohq/buefy-next into buefy/buefy (i.e. Buefy v1.0.0) we will start planning on migrating Buefy (Vue3) to Bulma v1.

Please let me know if you have any further questions
